tensorflow/compiler/xla/tests/exhaustive_unary_test.cc

```diff
@@ -369,7 +369,8 @@ class Exhaustive32BitOrLessUnaryTest
   // type being tested.
   template <typename T>
   void FillInput(Literal* input_literal) {
-    using IntegralT = typename IntegralTypeWithByteWidth<sizeof(T)>::type;
+    using IntegralT =
+    typename test_util::IntegralTypeWithByteWidth<sizeof(T)>::type;
```

Comment:
I realized that this is not use, we can replace the use of IntegralT in the routine with int64.

Reply:
ok. will do.

Reply:
are you sure this is true?  IntegralT is a type which depends on the template type T.  This does vary from use to use of FillInput.

Reply:
Yes, IntegralT was left here while it should be remove after I added  ConvertAndReplaceKnownIncorrectValueWith.

If input_val is a int64,  ConvertAndReplaceKnownIncorrectValueWith will convert it to IntegralT.

      IntegralT input_val = i + begin;
      input_arr[i] = ConvertAndReplaceKnownIncorrectValueWith<T>(input_val, 0);